From 9298116a4337c2c1c2ea6c4ba587783678e913ed Mon Sep 17 00:00:00 2001
From: Simon Ruggier <Simon80@gmail.com>
Date: Sun, 1 Apr 2012 23:41:28 -0400
Subject: [PATCH] Remove support for GL_ARB_sync

This works around a deadlock issue that World of Tanks started
triggering at version 7.2.
---
 dlls/wined3d/directx.c |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/dlls/wined3d/directx.c b/dlls/wined3d/directx.c
index 23cc6b2..c18bd1e 100644
--- a/dlls/wined3d/directx.c
+++ b/dlls/wined3d/directx.c
@@ -114,7 +114,6 @@ static const struct wined3d_extension_map gl_extension_map[] =
     {"GL_ARB_shader_texture_lod",           ARB_SHADER_TEXTURE_LOD,         0                           },
     {"GL_ARB_shading_language_100",         ARB_SHADING_LANGUAGE_100,       0                           },
     {"GL_ARB_shadow",                       ARB_SHADOW,                     0                           },
-    {"GL_ARB_sync",                         ARB_SYNC,                       0                           },
     {"GL_ARB_texture_border_clamp",         ARB_TEXTURE_BORDER_CLAMP,       0                           },
     {"GL_ARB_texture_compression",          ARB_TEXTURE_COMPRESSION,        0                           },
     {"GL_ARB_texture_compression_rgtc",     ARB_TEXTURE_COMPRESSION_RGTC,   0                           },
-- 
1.7.9.1
